src/gameobjects/graphics/Graphics.js

```diff
@@ -548,6 +548,110 @@ var Graphics = new Class({
         return this;
     },
 
+    /**
+     * Fill a rounded rectangle with the given position, size and radius.
+     *
+     * @method Phaser.GameObjects.Graphics#fillRoundedRect
+     * @since 3.11.0
+     *
+     * @param {number} x - The x coordinate of the top-left of the rectangle.
+     * @param {number} y - The y coordinate of the top-left of the rectangle.
+     * @param {number} width - The width of the rectangle.
+     * @param {number} height - The height of the rectangle.
+     * @param {number} [radius = 20] - The corner radius; It can also be an object to specify different radii for corners
+     * @param {number} [radius.tl = 0] Top left
+     * @param {number} [radius.tr = 0] Top right
+     * @param {number} [radius.br = 0] Bottom right
+     * @param {number} [radius.bl = 0] Bottom left
+     *
+     * @return {Phaser.GameObjects.Graphics} This Game Object.
+     */
+    fillRoundedRect: function (x, y, width, height, radius)
+    {
+        if (typeof radius === 'number')
+        {
+            radius = {tl: radius, tr: radius, br: radius, bl: radius};
+        }
+        else if (typeof radius === 'object')
+        {
+            var defaultRadius = {tl: 0, tr: 0, br: 0, bl: 0};
```

Comment:
Wondering if a loop is unnecessary overhead here? Direct assignment may be better: `radius.tl = radius.tl || 0`

Reply:
Yes, that would probably be better.